packages/wallet/.eslintignore

```diff
@@ -0,0 +1,11 @@
+/node_modules/
+/coverage/
+!.*
+/.vscode/
+/typings
+/migrations/
+**/*.json
+/website/
+/.etherlime-store/
+/lib/
+/dist/
```

Comment:
Many of these probably overlap with (a) `.gitignore` regexes and (b) other `.eslintignore` in other packages. Can we consolidate?